src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/CancellationReason.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.sscs.reference.data.mappings;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum CancellationReason {
+    WITHDRAWN("withdrawn", "Withdrawn", "", "BBA3"),
```

Comment:
This has been updated, look at:
https://tools.hmcts.net/confluence/display/CHMC/Ref+Data+-+Cancellation+Reasons